src/sph-partition.cc

```diff
@@ -125,12 +174,24 @@ void Worker::defineOptions(po::options_description & opts) {
          "to the output field name list if it isn't already included.")
         ("part.sub-chunk",
          po::value<std::string>()->default_value("subChunkId"),
-         "Sub-chunk ID output field name. This field field name is appended "
+         "Sub-chunk ID output field name. This field name is appended "
          "to the output field name list if it isn't already included.")
+        ("part.id",
+         po::value<std::string>(),
+         "Object ID input field name. This field name is optional. If it's provided "
```

Comment:
It confused me and I had to read it three times to understand. First time I read it as "parameter value is optional". Just say that parameter is optional, or don't say anything as all optional parameters are optional.

Reply:
Fixed that.

I should also mention that I'm not happy with the way this application is configured. Some values which are put into a config file can be also overwritten via the command line options. Though this may be good for debugging purposes, such extra flexibility would be bad in production where all aspects of the application configuration should be kept in a source-code control system (`git`, etc.) for proper bookkeeping and quality control. What one may still want to have in the command line options would be the names/locations of the input files, locations of the output products, site-specific configurations (hosts, ports, etc.), log level options, authentication options (in the future, assuming the input sources or output destinations were an object store in the Cloud environment. Anything else which affects data processing/formatting has to be in a configuration system (file).

This issue (among others) should be addressed in an expanded version of: https://jira.lsstcorp.org/browse/DM-26754